xbmc/FileItem.cpp

```diff
@@ -3766,6 +3766,22 @@ bool CFileItem::LoadDetails()
     return false;
   }
 
+  if (IsVideo())
+  {
+    if (HasVideoInfoTag())
+      return true;
+
+    CVideoDatabase db;
+    if (!db.Open())
```

Comment:
maybe log error here? I think this is pretty much unexpected

Reply:
Yes, can do.